docs/guides.md

```diff
@@ -1259,7 +1261,11 @@ To remove the installed certs in the pool:
 secureboot-certs clear
 ```
 
-Note that this does not remove the certs from the VMs or from disk. To remove them from disk, remove the ".auth" files for the certs you'd like to remove, on every host (found in `/var/lib/uefistored/`). In order to clear the certs from the VMs it is required to use `varstore-rm`. See [Remove Certificates from a VM](#remove-certificates-from-a-vm).
+:::tip
+Note that this does not remove the certs from the VMs nor from disk on XCP-ng 8.2.
+- On XCP-ng 8.2: to remove them from disk, remove the ".auth" files for the certs you'd like to remove, on every host (found in `/var/lib/uefistored/`).
```

Comment:
It would be good to explicitly state that on any version above 8.2.x, certs *will* be removed from disk on every host in the pool.